create_actions.rb

```diff
@@ -0,0 +1,83 @@
+def create_items(num)
+  case num
+  when 3
+    create_person
+  when 4
+    create_book
+  when 5
+    create_rental
+  end
+end
+
+def create_person
+  print 'Do you want to create a student(1) or a teacher(2)? [input a number]: '
+  choice = gets.chomp
+  print 'name: '
+  person_name = gets.chomp
+  print 'age: '
+  person_age = gets.chomp
+  case choice
+  when '1'
+    create_student(person_name, person_age)
+  else
+    create_teacher(person_name, person_age)
+  end
+end
+
+def create_student(name, age)
+  print 'Does student have parent\'s permission? [Y/N] '
+  ans = gets.chomp
+  case ans.downcase
+  when 'y'
+    student = Student.new('', age, name, parent_permission: true)
+    @app.people << student
+    @app.ids << student.id
+    puts 'Student created successfully!'
+  when 'n'
+    student = Student.new('', age, name, parent_permission: false)
+    @app.people << student
+    @app.ids << student.id
+    puts 'Student created successfully!'
+  else
+    print 'input not valid'
+  end
+end
+
+def create_teacher(name, age)
+  print 'what is your specialization? '
+  specialization = gets.chomp
+  teacher = Teacher.new(age, specialization, name)
+  @app.people << teacher
+  @app.ids << teacher.id
+  puts 'Teacher created successfully!'
+end
+
+def create_book
+  print 'title: '
+  title = gets.chomp
+  print 'author: '
+  author = gets.chomp
+  if title != '' && author != ''
+    book = Book.new(title, author)
+    @app.books << book
+    puts 'book created successfully'
+  else
+    puts 'invalid inputs'
+  end
+end
+
+def create_rental
+  print "please select the person who wants to rent (by postion on the list)\n"
+  list_people
+  person_no = gets.chomp.to_i
+  person = @app.people[person_no - 1]
+  print "\nplease select the book to be rented\n"
+  list_books
```

Comment:
- [Optional] Hi @tylher, did you notice that it's complicated to know the exact book to be selected?. 

![Screenshot from 2022-06-29 08-50-03](https://user-images.githubusercontent.com/63087491/176382393-528e23ab-f282-41cc-988a-3363046fcaf4.png)

- I guest it to be 1, and I'm correct :tada:  because I'm aware of it. What about another user who is about to use your app for the first time? :face_with_thermometer: . Also, let's consider the situation where so many books have been created. I believe it won't be easy to select the book. I think it's better to numerate the books like this:
  
   ```
please select the book to be rented: 

   [ 1 ] Title: First Book, Author: FB 
```

- This will surely create a better user experience.

Reply:
noted @FrederickMih , I will improve on that now, thank you!!